homing_missiles.py

```diff
@@ -109,12 +109,22 @@ def draw_entities(self, screen):
         pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
         self.draw_friendly_projectiles(screen)
 
+    def determine_enemy_color(self, enemy):
+        enemy_type = enemy.get_type()
+        if enemy_type == "BasicGrunt":
+            return YELLOW
+        elif enemy_type == "NotSoBasicGrunt":
+            return WHITE
+
+        return RED
+
     def draw_target_tags(self, screen):
         for enemy in self.universe_.enemies():
+            enemy_color = self.determine_enemy_color(enemy)
```

Comment:
I think the draw functions if done well won't be repeats. They are repeats right now because EVERYTHING is squares, but if say we wanted to make enemies triangles, then they would not look so similar.